Client.xcodeproj/project.pbxproj

```diff
@@ -14118,14 +14118,6 @@
 				version = 1.18.0;
 			};
 		};
-		1BE9ECAA271765C800D729BB /* XCRemoteSwiftPackageReference "rust-components-swift" */ = {
-			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/mozilla/rust-components-swift";
-			requirement = {
-				kind = exactVersion;
-				version = 91.0.1;
-			};
-		};
```

Comment:
@tarikeshaq do we no longer need rust components or I am assuming this is more of cleanup to support 93.0.4 on line 14150 🤔

Reply:
A cleanup! Some context in https://github.com/mozilla-mobile/firefox-ios/pull/10297#discussion_r838808335 

Looks like there were duplicate entries for `rust-component-swift`, and when updating using the Xcode UI this one didn't get bumped at all 😕 I got rid of it and firefox was built for me locally OK, I'm not sure why Xcode didn't automatically do the cleanup on its own in the past

Reply:
I feel this is weird github indentation because this seems fine on actual file

<img width="911" alt="Screen Shot 2022-04-29 at 1 01 09 PM" src="https://user-images.githubusercontent.com/8919439/165990187-c963409c-89b3-4cc0-a902-30aa9f5e35e0.png">
s